examples/Factorial/project.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Factorial",
+  "description": "Common routines for Factorial example",
+  "dependencies": {
+    "DotNetty.Common": {
+      "target": "project"
+    },
+    "DotNetty.Buffers": {
+      "target": "project"
+    },
+    "DotNetty.Transport": {
+      "target": "project"
+    },
+    "DotNetty.Codecs": {
+      "target": "project"
+    }
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "version": "1.0.0-*",
+          "type": "platform"
+        }
+      },
+      "imports": "dnxcore50"
+    }
```

Comment:
I couldn't figure out how to make `net451` work here due to `BigInteger`. Adding dependency on `System.Numerics` won't work, with dependency on `System.Runtime.Numerics` still fails to build `net451`. cc @terrajobst,